values/prometheus-operator/prometheus-operator-team.gotmpl

```diff
@@ -1,5 +1,3 @@
-{{- $teamId := .Release.Labels.team }}
```

Comment:
Why do you remove fix that was recently merged?

Reply:
by accident. Fix has been restored